src/systems/link.js

```diff
@@ -0,0 +1,16 @@
+var registerSystem = require('../core/system').registerSystem;
+
+/**
+ * Tracked controls system.
+ * It maintains a list with the available tracked controllers
+ */
+module.exports.System = registerSystem('link', {
```

Comment:
I don't think this needs to be a system, it doesn't provide any services to the link component. It can be part of the scene module, which is where all the other WebVR-related handlers live.